docs/source/usage/generating_metaframes.rst

```diff
@@ -91,30 +106,29 @@ spec
     The following code to achieve this would look like:
 
     .. code-block:: python
-        
+
         from metasyn.distribution import FakerDistribution, DiscreteUniformDistribution, RegexDistribution
+        from metasyn.config import VarConfig, DistributionSpec
 
         # Create a specification dictionary for generating synthetic data
-        var_spec = {
-
+        var_specs = [
```

Comment:
Yes, it is more wordy. However, personally I think it is better and less error-prone this way. So yes, I think we could provide a dictionary, but wasn't a big part of the point that you could get help from your IDE/type checker to make sure that you put it in correctly? Also, I saw that you figured out that we don't do dictionaries anymore below, so it will be more wordy regardless of supplying a list of dicts instead of a list of VarConfig's.